src/App.jsx

```diff
@@ -40,6 +41,7 @@ function App() {
               <Route exact path="/contacts" />
               <Route exact path="/map" />
               <Route exact path="/logs" element={<AdminPage />} />
+              <Route exact path="/admin-dashboard" element={<AdminDashboardPage />} />
```

Comment:
You can do this now or later, but dashboards (both admin and volunteer) should be on the `/` routes and be conditionally rendered based on the user role/what view they're in

Reply:
Sounds good! The new commit should address this! but lmk if there are still problems